pkg/document/json/rga_tree_split.go

```diff
@@ -426,16 +456,18 @@ func (s *RGATreeSplit) deleteNodes(
 
 		if node.Remove(editedAt, latestCreatedAt) {
 			s.treeByIndex.Splay(node.indexNode)
-
 			latestCreatedAt := createdAtMapByActor[actorIDHex]
 			createdAt := node.id.createdAt
 			if latestCreatedAt == nil || createdAt.After(latestCreatedAt) {
 				createdAtMapByActor[actorIDHex] = createdAt
 			}
+
+			key := createRemovedNodeMapKey(createdAt.Key(), node.id.offset)
+			removedNodeMap[key] = node
```

Comment:
It was confirmed that the key can be created with the `created_at` and `offset` properties of the node's id. How about generating the key with node id?

```suggestion
			removedNodeMap[node.id.key()] = node
```

Reply:
I think this is a really cool idea! I did this in the 1eefbb3 commit.
Removed the removedNodeMapKey type that I added, and it looks cooler. Thank you.